tools/python/categories_merge_language.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import sys
+import os
+
+if len(sys.argv) < 2:
+    print('Merges some language in categories.txt with English')
```

Comment:
Because it has only two positional parameters and no options. This is much simpler to read as a user and program as a developer.